### PR TITLE
Cut, Copy, Paste of file nodes + small refactor

### PR DESCRIPTION
**Pull request by DivergentEuropeans**
Extension of @miteshgoplani 's work in https://github.com/zowe/zlux-file-explorer/pull/84

- spelling
- You can now cut, copy, & paste file nodes into directories/currently active directory
- Fixed onPathChanged event emitter to duplicate less and fire for MVS too
- Removed deprecated emitters
- Removed deprecated code
- Re-ordered context menu options for more familiarity
- Added more descriptive error messages to not obvious errors

### CODE REVIEW

File: src/app/components/filebrowsermvs/filebrowsermvs.component.ts [hunk 1]
```diff
@@ -296,10 +296,6 @@ export class FileBrowserMVSComponent implements OnInit, OnDestroy {//IFileBrowse
     this.dialog.open(DatasetPropertiesModal, filePropConfig);
   }
 
-  browsePath(path: string): void{
-    this.path = path;
-  }
-
```
Comment (DivergentEuropeans) [inline src/app/components/filebrowsermvs/filebrowsermvs.component.ts:302]: not used

File: src/app/components/filebrowseruss/filebrowseruss.component.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
       placeholder="Enter an absolute path..."  
       [ngStyle] = "inputStyle" 
       class="filebrowseruss-search-input"
-      (keydown.enter)="displayTree(path, false); onPathChanged(path);"
```
Comment (DivergentEuropeans) [inline src/app/components/filebrowseruss/filebrowseruss.component.html:29]: duplicate firing

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 3]
```diff
@@ -147,10 +148,6 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     }
   }
 
-  browsePath(path: string): void {
-    this.path = path;
-  }
-
```
Comment (DivergentEuropeans) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:153]: not used

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 4]
```diff
@@ -392,16 +542,9 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
...
-  onClick($event: any): void {
-  }
-
-  onCopyClick($event: any): void {
-    this.copyClick.emit($event);
-  }
-
-  onNewFileClick($event: any): void {
-    this.newFileClick.emit($event);
-  }
```
Comment (DivergentEuropeans) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:404]: not used

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 5]
```diff
@@ -765,17 +907,6 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
...
-    this.log.debug('rename:' + this.selectedItem);
-    this.ussSrv.renameFile(this.selectedItem, this.checkPath(this.newPath))
-      .subscribe(
-        resp => {
-          this.updateUss(this.path);
-        },
-        error => this.errorMessage = <any>error
-      );
-  }
-
```
Comment (DivergentEuropeans) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:778]: rename done elsewhere, this deprecated

File: src/app/components/tree/tree.component.ts [hunk 6]
```diff
@@ -90,49 +90,7 @@ export class TreeComponent implements AfterContentInit, OnDestroy {
...
-    //             exceptionMessage: errorMessage.exceptionMessage
-    //           });
-    //           this.needCheckConnection.emit();
-    //         }
-    //       );
-    //     }
-    //   }
-    //   this.nodeExpanded.emit(true);
-    // }
-  }
```
Comment (DivergentEuropeans) [inline src/app/components/tree/tree.component.ts:135]: This is very old and these problems are now all solved, as we have lazy loading with errors and loading indicators

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**